### PR TITLE
W-15287172 Adding Retrieve Results action step

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+== Outbound Variables
+
+* *Status*
+* *JSON Result*
+
+[[wizard]]
+== Wizard
+
+Specify the *Execution ID* to query and the corresponding IDP *Action* and *Action Version*. These lists populate with the available document actions that your organization has published in IDP.
```
Comment (ms-tlu) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc]: **Nitpick (non-blocking)**: I think we should emphasize that the Action and Version must exactly match the configuration of the Submit AS which generated the Execution ID.

**Review by ms-tlu — APPROVED** [review]

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc (hunk 1 above)
Comment (Cristian-Venticinque) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc]:
```suggestion
Specify a combination of *Execution ID*, *Action*, and *Action Version* to query the results of that execution. These lists populate with the available document actions that your organization has published in IDP. Ensure you specify the same *Action* and *Action Version* used in the Submit Document to MuleSoft IDP configuration that generated the *Execution ID* you are checking.
```

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
+= Retrieve Results from MuleSoft IDP 
+
+Retrieve the results of a document processed with MuleSoft IDP. To run this action step you must provide an Execution ID, which generates during the execution of the Submit Document to MuleSoft IDP action step. The Execution ID links this action step with a document action execution to define which results to retrieve.
```
Comment (hannanelson) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc]:
```suggestion
Retrieve the results of a document processed with MuleSoft IDP. To run this action step, provide an Execution ID, generated during the execution of the Submit Document to MuleSoft IDP action step. The Execution ID links this action step with a document action execution to define which results to retrieve. 
```

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+* *Status*
+* *JSON Result*
+
+[[wizard]]
+== Wizard
+
+Specify a combination of *Execution ID*, *Action*, and *Action Version* to query the results of that execution. These lists populate with the available document actions that your organization has published in IDP. Ensure you specify the same *Action* and *Action Version* used in the Submit Document to MuleSoft IDP configuration that generated the *Execution ID* you are checking.
+
+Then, select the connected app credentials to use when connecting to IDP. You can use Pin credentials that you have configured or you can specify the *Client ID* and *Client Secret* manually in the wizard.
```
Comment (hannanelson) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc]:
```suggestion
Then, select the connected app credentials to use when connecting to IDP. You can use Pin credentials that you configured or specify the *Client ID* and *Client Secret* manually in the wizard. 
```

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp.adoc [hunk 4]
```diff
@@ -2,6 +2,22 @@
...
 
+Before you use these action steps, ensure you have created and published _document actions_ in MuleSoft IDP. Then, you can use the MuleSoft IDP action steps to submit documents for processing and retrieve the results of the execution by providing the generated Execution ID.
+
+include::partial$mulesoft-idp.adoc[tag=creditsWarning]
+
+* xref:toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc[]
++
+Retrieve the results of an IDP document action execution. 
++
+Before you can query the results, you must run the _Submit Document to MuleSoft IDP_ action step to trigger the execution of said document action.
```
Comment (hannanelson) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp.adoc]: font ok?

File: rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp.adoc [hunk 5]
```diff
@@ -2,6 +2,22 @@
...
+Retrieve the results of an IDP document action execution. 
++
+Before you can query the results, you must run the _Submit Document to MuleSoft IDP_ action step to trigger the execution of said document action. 
 * xref:toolbox-mulesoft-idp-submit-document-to-mulesoft-idp.adoc[]
 +
-Submit a document to MuleSoft IDP and execute a published document action to process the document. 
\ No newline at end of file
+Submit a document to MuleSoft IDP and execute a published document action to process the document. 
++
+Use the returned Execution ID to query the results later using the _Retrieve Results from MuleSoft IDP_ action step.
```
Comment (hannanelson) [inline rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp.adoc]: font ok? Previously, action step names were in regular font.

**Pull request merged**